layouts/shortcodes/observability_pipelines/configure_existing_pipelines/install_worker/amazon_eks.md

```diff
@@ -0,0 +1,30 @@
+1. Download the [Helm chart values file][4001] for AWS EKS.
+1. Click **Select API key** to choose the Datadog API key you want to use.
+1. Update Datadog Helm chart to the latest version:
+    ```shell
+    helm repo update
+    ```
+1. Run the command provided in the UI to install the Worker. The command is automatically populated with the environment variables you entered earlier.   
```

Comment:
It seems as though this item can be deleted, a command is not included here and the item below it seems to do the same thing?

Reply:
Thanks for catching this! Yes, I'll remove.